src/vm-repair/azext_vm_repair/repair_utils.py

```diff
@@ -427,12 +427,14 @@ def _unlock_singlepass_encrypted_disk_fallback(source_vm, resource_group_name, r
         volume_type = 'ALL'
 
     try:
+
+        # Set command as SINGLE_WITHOUT_KEK by default
```

Comment:
This seems to be changing functionality. Why is this edit here?

Reply:
Pylint complained the error was generic. the code that calls this has a catch for this type of exception.

Reply:
This code isn't even called, and this edit will simplify the script creation. Makes sense.